src/content/docs/implementation/setup/content-delivery-network.mdx

```diff
@@ -487,7 +556,7 @@ location: https://www.aemshop.net/
 
 To validate the APEX redirect, observe the location header returned by a request to the domain without `www`.
```

Comment:
All of these bash scripts EXCEPT for this APEX redirect snippet need to be updated with `https://www.aemshop.net` (not just https://aemshop.net).

APEX redirect intentionally does not have it, to show the redirect/location header. Other requests now need it to return same result, otherwise they will hit the redirect rule and not reflect accurate response.